20-minimal/Dockerfile.c9s

```diff
@@ -0,0 +1,72 @@
+FROM ubi9/ubi-minimal
+
+# FIXME:
+ADD https://gitlab.cee.redhat.com/platform-eng-core-services/internal-repos/-/raw/master/centos/c9s/c9s-compose.repo /etc/yum.repos.d/c9s-compose.repo
```

Comment:
This probably will not work. CentOS Stream 9 guest does not have access to internal infrastructure, like `https://gitlab.cee.redhat.com`

Reply:
I think the packages are available on mirrors now, so I will remove this line soon.